blog/2021-07-20-biweekly-updates-11.md

```diff
@@ -0,0 +1,67 @@
+---
+cat: update
+path: '/blog/biweekly-updates-11/'
+slug: 'biweekly-updates-11'
+publishedAt: '2021-07-20'
+author: Fábio Ferreira [Skyborg] 
+target: all
+title: 'Bi-Weekly Updates #11'
+teaser: |
+  Animeshon is growing, basic functionality achieved in a lot of services.
+overline: >
+  Bi-Weekly Updates
+---
+
+Ohayo, Skyborg \[Fábio\] desu. 
+
+
+I had the awesome opportunity to do an internship at Animeshon, so now I’m formally part of this project as well! It’s been a great experience, no complaints yet.  
+I’ll keep protesting about late biweekly updates, so don’t worry, I’m still on your side!
+
+
+First things first, to answer what everyone wants to know… yes, Matteo's english is even worse in the private repos. 
+I’ll start a small piggy bank to get him a dictionary on Christmas. Not for him to read, but for Christian to smack him with it every time he makes one of these: 
+>feat: ReconcileOracoloFulltextIndexes **acepts** force as **paramenter** to force reindex of **entieies** found in storage.
+
+
+### Ok, some updates on things that matter now:
+
+
+From my jurisdiction, the beautiful tracker! Well, if the name isn’t enough, I’ll give you a small explanation. 
+The tracker service / API is responsible for remembering what content you have seen, and when. It will be where the fact that you re-watched [Darling in the FranXX](https://animeshon.com/e/Darling_in_the_FranXX-Anime-YS-5VRlRF6Is) episode 15 482 times will be recorded, so that you can reminisce about all the happy time you spent seeing best girl.
+
+![Jumpy best girl](https://thumbs.gfycat.com/UntimelyBothHoneycreeper-size_restricted.gif)
```

Comment:
Please move the image to https://github.com/animeshon/template-homepage/tree/main/public/blog

EXTERNALLY HOSTED IMAGES BAD 😠😠😠